CHANGELOG.md

```diff
@@ -3,6 +3,12 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 
 # Change Log
 
+## [Unreleased] - ReleaseDate
```

Comment:
We don't directly edit this file, please see [this](https://github.com/nix-rust/nix/blob/master/CONTRIBUTING.md#pull-requests) on how to add a changelog :)